### ISSUE TITLE
Interest in a local/dev mode for small jobs or development

### ISSUE THREAD

**Issue by thenomemac** [stitchfix/flotilla-os#82]
Not sure if an issue is the right place to discuss this but... I'm thinking about forking this project and adding a local mode that devs could use to test flottila or you could use on prem if you're team has a huge vm. My vision for local mode if the following:

- Don't over opitimize for reliability, use AWS for that use case, these are batch jobs. If it crashes then hopefully you had a scheduler or Cron thatll know to retry again when the go process is back up
- minimize dependencies backing the AWS services into simple golang in process equivalents. 
- use local disk for storage. 
- use local docker deamon for execution. 
- support some form of artificial resource limiting to prevent local machine from becoming over provisioned. 

Does this sound interesting to anyone else's use case?

**Comment by thedatachef** [comment]
Yes - this local mode could be local execution engine, local queue manager, etc; happy to discuss

**Comment by thenomemac** [comment]
Started working on this yesterday but I'm a golang noob so some of the implementation was more involved than I thought at first. Anyways. I'll probably work on this more time permitting. Very interested in this project.

**Comment by 2morales** [comment]
Hello,

Updates on this? It would be a really useful feature for people having their own infrastructure.

Thank you

**Comment by thenomemac** [comment]
I have not had time to work on this.

**Comment by 2morales** [comment]
Okay, thank you!
